homeassistant/components/sensor/const.py

```diff
@@ -138,6 +138,12 @@ class SensorDeviceClass(StrEnum):
     Unit of measurement: `A`, `mA`
     """
 
+    CURRENT_FLOW = "current_flow"
```

Comment:
Adding a device class requires an architecture discussion approval.

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

I suggest removing this change from this PR to not block the PR.

Reply:
Ok, I can do that. Current flow (mainly amp-hours) are fairly commonly used in the boat/RV space but probably not widely used elsewhere (in favor of power/watts).

Reply:
The latest commit removes these changes to sensor/const.py.